README.md

```diff
@@ -65,6 +65,10 @@ $ docstr-coverage some_project/src
 	* 1 - Print overall statistics
 	* 2 - Also print individual statistics for each file
 	* 3 - Also print missing docstrings (function names, class names, etc.)
+* *--docstr-ignore-file - Filepath to list of patterns to ignrore. Patterns are file-pattern name-pattern pairs. File content example:
```

Comment:
This description seems a bit vague to me. I think the way you explained the feature in the PR comment was more informative. In any case, I think some more detail would help a lot here and in the file content example immediately below, especially since your new feature is quite specific and a bit more complicated than the other filters. 

Let me know what you think 

```suggestion
* *--docstr-ignore-file - Filepath to list of patterns to ignore. Patterns are file-pattern name-pattern pairs. File content example:
```